guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,388 @@
+   retina = "/assets/images/guide/how-it-works/silent-payments/silent-payments-header@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/silent-payments-header-mobile.png"
+   mobileRetina = "/assets/images/guide/daily-spending-wallet/silent-payments/silent-payments-header-mobile@2x.png"
+   alt-text = "A header image with a purple gradient, the text Silent Payments and a network visual."
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Silent payments
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+[Silent payments](https://github.com/bitcoin/bips/blob/master/bip-0352.mediawiki) is a protocol involving static addresses that are used to derive unique on-chain address during every transaction. This prevents [address reuse](/guide/glossary/address/#address-reuse) without  repeated user interaction, enabling an interaction model for on-chain transactions centred around users.
+
+For eg: Alice, who runs an NGO, can simply post a static address on her website, and receive bitcoin donations at unique on-chain addresses that only she can identify. The static address itself never shows up on-chain.
+
+Silent payments also allow users to customize their static address with labels that are detected when payments are received. This is a powerful feature providing useful, actionable information in [coin control](/guide/how-it-works/silent-payments/) and [contacts](/guide/daily-spending-wallet/contacts/) while reducing manual efforts.
+
+
+## How silent payments work
+
+Using the same address for multiple payments tells everybody on the network that these payments were made to the same user. Every interaction to get a new address takes time and effort, along with the possibility of mistakes while handling on-chain addresses. Silent payments circumvent both these issues by using static addresses.
+
+Here's how silent payments make this possible:
+1. The receiver shares/publishes a static payment address
+2. The sender obtains & uses it to derive a unique on-chain address
+3. The sender broadcasts a transaction that pays this derived address
+4. The receiver identifies the payment as theirs by verifying that they control this address
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work.png"
+   mobileRetina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   alt-text = "An image that illustrates how silent payments work in 4 steps."
+   caption = "."
+   width = 1024
+   height = 522
+   layout = "full-width"
+%}
+
+Silent payments introduces a new model based on new primitives/concepts that separates payment information based on its use. Below is a brief summary:
```

Comment:
This sentence sounds quite cryptic. In addition, it's a relatively abrupt jump from the section that explains how SP work. 

How about adding a new section called "A new conceptual model". The new section could start with something like the sentence below:

_Silent payments introduce a new conceptual model which is based on people, rather than addresses. This new model unlocks opportunities for more contextual and intuitive user experiences for bitcoin payments._

Reply:
Yup, this one being a bit naughty and seems to be an issue despite some efforts :/

I like your conceptual model section idea, especially coz I added a second table just yesterday! Will revisit.

Reply:
Phrasing has been improved in myriad ways based on this feedback as well as reading session.